src/game.cpp

```diff
@@ -117,6 +119,6 @@ void Game::usePiece(Piece* piece) {
 }
 
 void Game::drawScreen() {
-	this->currPiece->draw();
-	this->board.draw();
+	this->currPiece->draw(window);
```

Comment:
do you really need to draw both the piece and the board? Isn't drawing the board sufficient?

Reply:
Yes.